src/explorer.ts

```diff
@@ -195,6 +196,40 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
     }
 }
 
+class KubernetesNodeFolder extends KubernetesResourceFolder {
+    constructor() {
+        super(kuberesources.allKinds.node);
+    }
+
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        const nodes = await kubectlUtils.getGlobalResources(kubectl, 'nodes');
+        return nodes.map((node) => new KubernetesNodeResource(node.name, node));
+    }
+}
+
+class KubernetesNodeResource extends KubernetesResource {
+    constructor(name: string, meta: any) {
+        super(kuberesources.allKinds.node, name, meta);
+    }
+
+    async getTreeItem(): Promise<vscode.TreeItem> {
+        const treeItem = await super.getTreeItem();
+        treeItem.collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
+        return treeItem;
+    }
+
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        const pods = await kubectlUtils.getPods(kubectl, null, 'all');
+        const filteredPods = [];
+        pods.map((p) => {
```

Comment:
Can this use `filter` instead of a `map` with a side-effectful `push`?  I think that would be more self-documenting...

Reply:
done.